index.js

```diff
@@ -42,6 +46,7 @@ module.exports = {
     "max-line-length": [ 80, {
       ignore: "non-comments",
     } ],
+    "max-nesting-depth": 3,
```

Comment:
Ah I see. Well I was hoping that between this and `bem-linter` we could cover all the bases. I guess see how they work together when ready.